decam/transmission_system/n1/Y_DECam_c0005_10095.0_1130.0/2012-09-12T00:00:00.ecsv

```diff
@@ -0,0 +1,169 @@
+# %ECSV 1.0
+# ---
+# datatype:
+# - {name: wavelength, unit: Angstrom, datatype: float64}
+# - {name: throughput, datatype: float64}
+# meta: !!omap
+# - {MODE: DETECTOR}
+# - {TYPE: transmission_system}
+# - {CALIBDATE: '2012-09-12T00:00:00'}
+# - {INTRUME: DECam}
+# - {OBSTYPE: transmission_system}
+# - {CALIBCLS: lsst.ip.isr.IntermediateTransmissionCurve}
```

Comment:
This will need to use the subclasses that have the appropriate OBSTYPE value set (`lsst.ip.isr.IntermediateSystemTransmissionCurve` in this case.
The directory tree matches, although the filters may need to be lowercase.
Adding `CALIB_CREATION_DATE` and `CALIB_CREATION_TIME` metadata entries will ensure that any exposures processed with this have those values set in the header (after DM-37129)

Reply:
Are you saying that the `physical_filter` should be lowercased?  That seems surprising.  Also, should the `CALIB_CREATION_DATE` and `CALIB_CREATION_TIME` be when this is run, or the same as `CALIBDATE`?  Or some other random time close to when they were generated for DES?

Reply:
The filters-as-directory names need to be lower cased.  This is where the filter is converted to expected directory name: https://github.com/lsst/obs_base/blob/main/python/lsst/obs/base/_read_curated_calibs.py#L237 , with an additional " " -> "_" translation after a note on previous ticket mentioning DECam filters specifically.
The `CREATION` keywords are used as a "cheap way" to distinguish calibrations (it's been one of the things I check to make sure I'm using the correct calib, and added it to the headers in DM-37129 to do the same there).  I think I set the LATISS throughputs to be the approximate time I finished editing them, so they're not fully random, but the main point is that I can check that those are the 2022 version of the curves.